### PR TITLE
Display opening hours when JS doesn't fire

### PR DESCRIPTION
**Pull request by davidpmccormick**
## What is this PR trying to achieve?

Making sure opening hours display nicely when JS isn't working (part of #193).

## What does it look like?

screen shot 2017-01-10 at 17 38 32

### CODE REVIEW

File: client/scss/components/_opening_hours.scss [hunk 1]
```diff
@@ -59,12 +59,15 @@
   @include font('HNL6');
   color: color('accessible-grey-2');
   width: 100%;
-  display: flex;
+  display: table;
```
Comment (jamesgorrie) [inline client/scss/components/_opening_hours.scss]: Does this tell screen readers to read it as table data?
And does setting it as flex do the opposite?

Why isn't this table by default as it's tables?

File: server/views/components/opening-hours/index.njk [hunk 2]
```diff
@@ -7,10 +7,10 @@
     {% endfor %}
   </ul>
   {% for place in places %}
-    <table id="panel-{{ place.id }}"class="opening-hours__table js-tabpanel">
+    <table id="panel-{{ place.id }}" class="opening-hours__table js-tabpanel">
       <caption id="{{ place.id }}" class="opening-hours__caption js-tabfocus">{{ place.name }}</caption>
       <thead class="opening-hours__thead">
-        <tr>
+        <tr class="opening-hours__tr">
```
Comment (jamesgorrie) [inline server/views/components/opening-hours/index.njk]: We've added this class - but not in the CSS?

File: client/scss/components/_opening_hours.scss (hunk 1 above)
Comment (davidpmccormick) [inline client/scss/components/_opening_hours.scss]: > Does this tell screen readers to read it as table data?
And does setting it as flex do the opposite?

As far as I can tell, these display properties will have no bearing on what is read by screenreaders; only visual appearance will be affected.

> Why isn't this table by default as it's tables?

Good point. Removed.

File: server/views/components/opening-hours/index.njk (hunk 2 above)
Comment (davidpmccormick) [inline server/views/components/opening-hours/index.njk]: This class pre-exists in the CSS.

**Pull request merged**